opendevin/server/session.py

```diff
@@ -89,13 +89,23 @@ async def start_listening(self):
                     await self.create_controller(data)
                 elif action == "start":
                     await self.start_task(data)
+                elif action == "files":
```

Comment:
IMO this should be a REST API, like our `/litellm-models` endpoint. We don't really need to stream it